src/mscorlib/src/System/Threading/Tasks/Task.cs

```diff
@@ -1617,21 +1617,8 @@ internal TaskScheduler ExecutingTaskScheduler
         /// </remarks>
         public static TaskFactory Factory { get { return s_factory; } }
```

Comment:
Nit - it may be nice to change this one to use the same syntax as CompletedTask to make the style locally consistent.